package.json

```diff
@@ -22,6 +22,7 @@
     "danger": "^10.0.0"
   },
   "dependencies": {
+    "cors": "^2.8.5",
```

Comment:
What is this CORS dependency for?

Reply:
It's for the backend since it's at port 9000